docs/en/observability/apm/act-on-data/apm-alerts.asciidoc

```diff
@@ -1,5 +1,10 @@
+////
+TO DO:
+Should we move this to https://www.elastic.co/guide/en/observability/master/create-alerts.html ?
+////
```

Comment:
Yes. Then the question is what to do with custom links. Custom links are also incident management, so I can see those moving to the incident management section too.

Reply:
@bmorelli25 what do you think about these two options:

### Option 1
Keep the "Act on data" page. Mention APM alerts/rules and link to the "Incident management" section. Include the custom links content directly on the "Act on data" page.

_Why?_ As far as I can tell, custom links are specific to APM (at least within Observability).
 
<img width="1522" alt="Screenshot 2024-09-16 at 12 30 38 PM" src="https://github.com/user-attachments/assets/801961db-c984-44b4-9448-600ba649b9ee">

### Option 2
Get rid of the "Act on data" page. Move all content to "Incident management" section.

_Why?_ Custom links are also incident management.

<img width="1518" alt="Screenshot 2024-09-16 at 12 40 13 PM" src="https://github.com/user-attachments/assets/86a689e1-d09a-4227-94bb-9ccbcc68e8cc">